src/blister.c

```diff
@@ -919,7 +919,7 @@ void PlainBlist (
     /* replace the bits by 'True' or 'False' as the case may be            */
     /* this must of course be done from the end of the list backwards      */
     for ( i = len; 0 < i; i-- )
-        SET_ELM_PLIST( list, i, ELM_BLIST( list, i ) );
+        SET_ELM_PLIST(list, i, ELM_BLIST_UNSAFE(list, i));
```

Comment:
Hmm, so that means the previous commit actually results in a broken GAP, right?

So normally I would say this commit needs to be squashed into the preceding commit (the two commits are not independent from each other). (I'll survive and won't complain loudly if this is not done, but it's how I would do it)